CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Contributing to ooni-probe
+
+ooni-probe is a Free and Open Source software project that welcomes
+new contributions.
+
+## How to contribute code
+
+For guidelines on what coding style we follow refer to
+the [coding style document](/doc/coding-style.md).
```

Comment:
We actually have our coding style inside of HACKING. To  be honest I would suggest we refactor that to remove references to source code layout (since it's dated) and rename it to doc/coding-style.md